### ISSUE TITLE
Please explain how to use pyenv with uwsgi

### ISSUE THREAD

**Issue by COLABORATI** [yyuu/pyenv#598]
It would be very appreciated if you would like to explain how to use uwsgi with pyenv and also provide an example configuration for e.g. a simple flask app. I have some material about this from several weblogs, old tutorials etc, but it would be much preferable to have one "official" how-to by the pyenv author. Thanks!

**Comment by COLABORATI** [comment]
BTW also it would be great if you could motivate the uwsgi authors to add a chapter about how to integrate with pyenv, I can not find the string "pyenv" in their docs.

**Comment by COLABORATI** [comment]
https://github.com/unbit/uwsgi/issues/1251

**Comment by COLABORATI** [comment]
As there is unfortunately an unbelievable chaos of how to deploy python web apps with more then one zen way to do this, it would be great if there was also explanation for how to use pyenv with a system-installed uwsgi (via package manager) and how to use it with uwsgi installed in the pyenv local environment (installed via pip) - and which one is better? Also if there might be differences / problems between python2 and python3, this should be explained. THANK YOU!

**Comment by yyuu** [comment]
This is not your question forum, an issue tracker.

**Comment by TomGoBravo** [comment]
I struggled to get uwsgi to run my app, which worked really well in a pyenv virtualenv. I couldn't find docs either. So far just activating the virtualenv, running `pip install uwsgi` and putting the path of the uwsgi binary (with default setup found with `ls -l $HOME/.pyenv/versions/*/bin/uwsgi`) in my systemd Unit File (I'm following https://www.digitalocean.com/community/tutorials/how-to-serve-flask-applications-with-uswgi-and-nginx-on-ubuntu-18-04). I don't know if it makes any difference but I built the python version with `PYTHON_CONFIGURE_OPTS="--enable-shared" pyenv install 3.7.2`. It looks like python as started by this uwsgi binary uses the virtualenv without needing to set it in the uwsgi options.
